purchases/src/main/kotlin/com/revenuecat/purchases/common/logUtils.kt

```diff
@@ -7,34 +7,35 @@ import com.revenuecat.purchases.PurchasesErrorCode
 /**
  * Whether debug logs are enabled
  */
-val LogLevel.debugLogsEnabled: Boolean
+internal val LogLevel.debugLogsEnabled: Boolean
     get() = this <= LogLevel.DEBUG
 
-fun LogLevel.Companion.debugLogsEnabled(enabled: Boolean): LogLevel {
+internal fun LogLevel.Companion.debugLogsEnabled(enabled: Boolean): LogLevel {
     return if (enabled) {
         LogLevel.DEBUG
     } else {
         LogLevel.INFO
     }
 }
 
-fun verboseLog(message: String) {
+internal fun verboseLog(message: String) {
     logIfEnabled(LogLevel.VERBOSE, currentLogHandler::v, message)
 }
 
-fun debugLog(message: String) {
+internal fun debugLog(message: String) {
     logIfEnabled(LogLevel.DEBUG, currentLogHandler::d, message)
 }
 
-fun infoLog(message: String) {
+internal fun infoLog(message: String) {
     logIfEnabled(LogLevel.INFO, currentLogHandler::i, message)
 }
 
+// Public because it's been used in the hybrids
```

Comment:
Hmm this is so the hybrids use the same logging mechanism as the natives? I'm not sure if this is the correct way to do this... We might want to think about alternatives for this... Maybe we want the hybrids to have a completely different logging system, (though we would need to hook it up to the log handler in that case).

For now, I'm not a fan of exposing this... But maybe it's ok for the sake of moving this forward.

Reply:
I agree with you, that's why I didn't expose the other ones. I can make it internal, but it's going to require some changes in phc to use a customer handler as you mention. 

I vote for making it public now and refactor that in the future, so we don't slow down releasing new versions, but happy to change it

Reply:
Yup sounds good!